qutebrowser/html/history.html

```diff
@@ -16,43 +16,165 @@
     white-space: nowrap;
 }
 
-.date {
-    color: #888;
-    font-size: 14pt;
-    padding-left: 25px;
+table {
+    margin-bottom: 30px;
 }
 
-.pagination-link {
-    display: inline-block;
-    margin-bottom: 10px;
-    margin-top: 10px;
-    padding-right: 10px;
+.date {
+    color: #555;
+    font-size: 12pt;
+    padding-bottom: 15px;
+    font-weight: bold;
+    text-align: left;
 }
 
-.pagination-link > a {
-    color: #333;
-    font-weight: bold;
+.session-separator {
+    color: #aaa;
+    height: 40px;
+    text-align: center;
 }
+
 {% endblock %}
 
-{% block content %}
+{% block script %}
```

Comment:
What do you think about moving this into its own `history.js` file in `qutebrowser/javascript`? I think that'd be cleaner, but you'd need to add a simple `qute:javascript` handler to read the javascript - essentially something like what was added in 37fa7431b03842f55bd1d1cfe57a059aa9d7bfe5 (but reverted subsequently).

Note that when you do that, `eslint` (a javascript linter) will also start to chime in. It's not really fine-tuned yet, so if you don't agree with some of it's suggestions, let me know!